vue-app/src/components/FilterDropdown.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div class="category-filter" v-click-outside="closeDropdown">
+    <div
+      :class="{
+        'filter-btn': true,
+        'filter-btn-cart-closed':
+          !!$store.state.currentUser && !$store.state.showCartPanel,
+        'filter-btn-cart-open':
+          !!$store.state.currentUser && $store.state.showCartPanel,
```

Comment:
@samajammin @wackerow do you remember the purpose of these classes? it is setting `width: auto` in md resolution sizes. I think that the idea is to make the Filter fill the complete space in md sizes. Look at this:

Cart is closed:
![cart closed](https://user-images.githubusercontent.com/468158/123272383-09defb00-d545-11eb-8f4c-1f1825fa79c7.png)

Cart is open:
![cart open](https://user-images.githubusercontent.com/468158/123272416-106d7280-d545-11eb-952f-6d4270ac2213.png)

As you can see, when the Cart is closed, the Filter dropdown does not expand to fill all the space but when the Cart is opened it does. I guess we want to make it consistent.

Reply:
As we discussed, I'll keep it with the compact version in both states.

Reply:
Sorry for delay in responding to this, but yeah `filter-btn-cart-open` and `filter-btn-cart-closed` are there to conditionally lay out the filter menu based on a combination of both screen width (media queries) and weather or not the cart panel is visible / open or closed `state.showCartPanel`. 

This was a work around since there was no easy way to do a "media query" for the parent container size. The layout of the main content shifts slightly depending if there is no user logged in (thus no cart at all), a user logged in with a closed cart (slight shift to accomodate cart button) and an open cart (biggest shift). Hope this helps clarify.

Reply:
Thanks for the explanation. 

I have removed them. Now that we are showing the filter always in the compact version I think there is no need for these classes anymore.